kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/PodOperationsImpl.java

```diff
@@ -138,29 +139,40 @@ public PodOperationContext getContext() {
     return (PodOperationContext) context;
   }
   protected String getLogParameters() {
+    HttpUrl.Builder httpUrlBuilder = new HttpUrl.Builder();
     StringBuilder sb = new StringBuilder();
```

Comment:
Is this still WIP? Why are you keeping `StringBuilder` too? I don't see it getting used anywhere.

Reply:
forgot to remove it .sorry

Reply:
I've rolled back to using stringbuilder here only.. since httpurl isn't an ideal replacement for stringbuilder in this method.